lib/Doctrine/ORM/Query/AST/Functions/AbsFunction.php

```diff
@@ -16,7 +16,7 @@
  */
 class AbsFunction extends FunctionNode
 {
-    /** @var SimpleArithmeticExpression */
+    /** @var SimpleArithmeticExpression|ArithmeticTerm|PathExpression */
```

Comment:
The common ancestor would be `Node`… what should happen to the comment on line 13 then, in both situations?